### PR TITLE
feat(layout): move header to separate component

### PR DESCRIPTION
**Pull request by dgutoryan**

### CODE REVIEW

File: apps/frontend/src/app/components/layout/app-layout.tsx [hunk 1]
```diff
@@ -1,52 +1,12 @@
...
-              placement="bottomRight"
-              overlay={userMenu}
-              trigger={['click']}
-            >
-              <Avatar style={{ cursor: 'pointer' }}>{firstLetter}</Avatar>
-            </Dropdown>
-          </Col>
-        </Row>
-      </Layout.Header>
+      <Header></Header>
```
Comment (IlyaRucavitcyn) [inline apps/frontend/src/app/components/layout/app-layout.tsx:9]: If this has no content - we can use single tag here (`<Header/>`)

File: apps/frontend/src/app/components/layout/header.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <Col>
+          <Menu
+            theme="dark"
+            mode="horizontal"
+            selectable={false}
+            disabledOverflow={true}
+            selectedKeys={[path]}
+          >
+            <Menu.Item key={RouteName.ACTIVITY}>
+              <Link to={RouteName.ACTIVITY}>Activity</Link>
```
Comment (IlyaRucavitcyn) [inline apps/frontend/src/app/components/layout/header.tsx:34]: Do we already have i18n in place? If yes - we need to translate this

**Review by IlyaRucavitcyn — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/components/layout/app-layout.tsx (hunk 1 above)
Comment (dgutoryan) [reply apps/frontend/src/app/components/layout/app-layout.tsx:9]: Done

File: apps/frontend/src/app/components/layout/header.tsx (hunk 2 above)
Comment (dgutoryan) [reply apps/frontend/src/app/components/layout/header.tsx:34]: No, we don't. Let's do it in the next MR

**Review by IlyaRucavitcyn — APPROVED** [review]

**Pull request merged**